frontend/src/scenes/notebooks/NotebookSelectButton/notebookSelectButtonLogic.ts

```diff
@@ -46,14 +53,25 @@ export const notebookSelectButtonLogic = kea<notebookSelectButtonLogicType>([
             actions.loadAllNotebooks()
             actions.loadNotebooksContainingResource()
         },
+        setCreatedBy: async (_, breakpoint) => {
+            // Debouncing similarly as the search query
+            await breakpoint(300)
+            actions.loadAllNotebooks()
+            actions.loadNotebooksContainingResource()
+        },
     })),
     loaders(({ props, values }) => ({
         allNotebooks: [
             [] as NotebookListItemType[],
             {
                 loadAllNotebooks: async (_, breakpoint) => {
                     await breakpoint(100)
-                    const response = await api.notebooks.list({ search: values.searchQuery || undefined })
+                    const response = await api.notebooks.list({
```

Comment:
Question: I don't see any UI indication that we truncate by 50, would it add value to have a tiny note somewhere at the bottom?